addons/longhorn/template/base/install.sh

```diff
@@ -119,3 +122,30 @@ function longhorn_preflight() {
     local src="${DIR}/addons/longhorn/${LONGHORN_VERSION}"
     echo "${src}/host-preflight.yaml"
 }
+
+function check_mount_propagation() {
+    local src=$1
+
+    kubectl get ns longhorn-system 2>/dev/null 1>/dev/null || kubectl create ns longhorn-system 1>/dev/null
+
+    render_yaml_file "$src/tmpl-mount-propagation.yaml" > "$src/mount-propagation.yaml"
+    kubectl create -f "$src/mount-propagation.yaml"
```

Comment:
what if delete fails and you re-run? should we delete before we create here?

Reply:
or maybe just have it be `apply -f`